index.html

```diff
@@ -63,114 +63,114 @@ <h3 class="secondaryText">
             <img src="./src/misc/Action.svg" alt="">
         </div>
     </section>
-    <section class="recentWorks centeredGrid">
+    <section class="recentWorks">
         <h2>
-            My Recent <br> Works
+            My Recent Works
```

Comment:
- [ ] Add more space between the home section and my recent works section (as you can see in the yellow color in the image below)

- [ ] Add min-width and max-width CSS property for your cards in My recent works to avoid issues when you are changing the window size.

![image](https://user-images.githubusercontent.com/60164239/130100603-c70fbc37-8d66-43d6-825a-30491644692e.png)